File: Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/requests/CreateEmployeeRequest.java
package com.elgin.elginexperience.shipay.requests;

import com.google.gson.annotations.SerializedName;

public class CreateEmployeeRequest {
    public CreateEmployeeRequest(String name, float salary, int age) {
        this.name = name;
        this.salary = salary;
        this.age = age;
    }
    @SerializedName("name")
    public String name;

    @SerializedName("salary")
    public float salary;

    @SerializedName("age")
    public int age;
}

File: M8/Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/requests/CreateEmployeeRequest.kt
package com.example.e1_elgin_kotlin.shipay.requests

import com.google.gson.annotations.SerializedName

data class CreateEmployeeRequest(
    @field:SerializedName("name") var name: String,
    @field:SerializedName("salary") var salary: Float,
    @field:SerializedName("age") var age: Int
)
